app/steps/ui/iht/paper/index.js

```diff
@@ -5,14 +5,15 @@ const validator = require('validator');
 const numeral = require('numeral');
 const FieldError = require('app/components/error');
 const {get} = require('lodash');
+const FeatureToggle = require('app/utils/FeatureToggle');
 
 class IhtPaper extends ValidationStep {
 
     static getUrl() {
         return '/iht-paper';
     }
 
-    handlePost(ctx, errors) {
+    handlePost(ctx, errors, formdata, session, hostname, featureToggles) {
```

Comment:
Can we have a unit test for at least the new variable that was added to this method